tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,199 @@
+                                         return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+lat_roi *= 1e3
+amp_roi *= 1e6
+
+# Print output
+print('Channel: eeg59')
+print(f'Peak Latency: {lat_roi:.3f} msec')
+print(f'Peak Amplitude: {amp_roi:.3f} \u00B5V')
+
+# %%
+# While the peak latency is the same in channels ``eeg55`` and ``eeg59``, the
+# peak amplitudes differ. The above approach can be done on virtual channels
+# created with the `~mne.channels.combine_channels` function and on difference
+# waves created with the `mne.combine_evoked` function (``aud_minus_vis``).
+#
+# While beyond the scope of this tutorial, peak measures are very susceptible
+# to high frequency noise (for discussion, see :footcite:`Luck2014`). One way
+# to avoid this is to apply a non-causal low-pass filters to the ERP. While
+# this can reduce bias in peak amplitude measures due to high frequency
+# noise, it can introduce challenges in interpreting latency measures for
+# effects of interest :footcite:`Rousselet2012,VanRullen2011`.
 #
+# Also, if using peak measures, it is critical to visually inspect the data to
+# make sure the selected time window actually contained something that looks
+# like a peak. It can be problematic if the time window is incorrect and, on
+# some participants, only identifies the rising edge of a peak.
+#
+# The following demonstrates why visual inspection is crucial. The dotted
+# lines depict the time window searched, and the solid lines indicates the
+# peak latency returned by `~mne.Evoked.get_peak`. The example uses ``eeg59``
+# with the time window defined previously (.065 to .115) and a time window
+# that occurs later in time and does not include the peak. As can be seen
+# with the red lines below (the bad time window), the returned peak is not
+# the true peak.
+
+# Get BAD peak measures
+bad_tmin, bad_tmax = .09, .12
+_, bad_lat_roi, bad_amp_roi = \
+    l_vis_roi.get_peak(mode='pos', tmin=bad_tmin, tmax=bad_tmax,
+                       return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+bad_lat_roi *= 1e3
+bad_amp_roi *= 1e6
+
+# Print output
+print('** BAD TIME WINDOW **')
+print('Channel: eeg59')
+print(f'Peak Latency: {bad_lat_roi:.3f} msec')
+print(f'Peak Amplitude: {bad_amp_roi:.3f} \u00B5V')
+
+# Make an empty figure handle and axis
+fig, ax = plt.subplots(nrows=1, ncols=1)
+
+# Plot the ERP. Note the time_unit is milliseconds
+l_vis_roi.plot(axes=ax, time_unit='ms', show=False,
+               titles='Good and Bad Peaks at eeg59')
+
+# Plot lines for good peak window
+hg = ax.axvline(lat_roi, linestyle='-', color='g')
+ax.axvline(tmin * 1e3, linestyle=':', color='g')
+ax.axvline(tmax * 1e3, linestyle=':', color='g')
+
+# Plot lines for bad peak window
+hb = ax.axvline(bad_lat_roi, linestyle='-', color='r')
+ax.axvline(bad_tmin * 1e3, linestyle=':', color='r')
+ax.axvline(bad_tmax * 1e3, linestyle=':', color='r')
```

Comment:
I would avoid red and green colors for colorblind people.

also I find the legend a bit hard to parse. It does not represent windows with areas
and the dotted lines are not explained by the legend. I would use
https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.axvspan.html
to represent time windows.

Reply:
Thanks for this suggestion. I will work on incorporating this sometime this weekend.

Reply:
I just pushed a commit to address the plot. I separated it into two sub-plots (to remove the confusing legend), used ``axvspan`` to show the time window, and changed the colors. I changed the text as well to try and make that bit of the example flow better. I did my best to catch any formatting/typos before pushing the comment, and will re-check when the docs render.